server/src/associate/associate.service.ts

```diff
@@ -59,7 +59,7 @@ export class AssociateService {
 			},
 		});
 
-		return check1 !== undefined || check2 !== undefined;
+		return !!check1 || !!check2;
```

Comment:
I feel like there has to be a better name for these variables. Idk what though

Reply:
I can make that clearer